cirq-google/cirq_google/calibration/phased_fsim.py

```diff
@@ -1050,38 +1050,17 @@ def try_convert_gate_to_fsim(gate: cirq.Gate) -> Optional[PhaseCalibratedFSimGat
         If provided gate is equivalent to some PhaseCalibratedFSimGate, returns that gate.
         Otherwise returns None.
     """
-    if cirq.is_parameterized(gate):
-        return None
-
     phi = 0.0
     theta = 0.0
     phase_exponent = 0.0
-    if isinstance(gate, SycamoreGate):
-        phi = np.pi / 6
-        theta = np.pi / 2
-    elif isinstance(gate, cirq.FSimGate):
-        theta = gate.theta
-        phi = gate.phi
-    elif isinstance(gate, cirq.ISwapPowGate):
-        if not np.isclose(np.exp(np.pi * 1j * gate.global_shift * gate.exponent), 1.0):
-            return None
-        theta = -gate.exponent * np.pi / 2
-    elif isinstance(gate, cirq.PhasedFSimGate):
-        if not np.isclose(gate.zeta, 0.0) or not np.isclose(gate.gamma, 0.0):
-            return None
-        theta = gate.theta
-        phi = gate.phi
-        phase_exponent = -gate.chi / (2 * np.pi)
-    elif isinstance(gate, cirq.PhasedISwapPowGate):
-        theta = -gate.exponent * np.pi / 2
-        phase_exponent = -gate.phase_exponent
-    elif isinstance(gate, cirq.ops.CZPowGate):
-        if not np.isclose(np.exp(np.pi * 1j * gate.global_shift * gate.exponent), 1.0):
-            return None
-        phi = -np.pi * gate.exponent
-    else:
+    cgate = cast(Optional[cirq.PhasedFSimGate], FSimGateFamily().convert(gate, cirq.PhasedFSimGate))
```

Comment:
I think it's a special purpose gate used for calibrations and we can always use the gate family to convert to `PhasedFSimGate` and then downgrade to `PhaseCalibratedFSimGate`, similar to what has been done now. 

Since there's only one usage of PhaseCalibratedFSimGate right now, I'd we can leave it as it is for now and add it to this gate family later if needed.